Add Port to Input Fild

superwinni2 opened the pull request:
Due to that you can choose on which Port to listen for input. Usefull if you don't run logstash witout root User.

superwinni2 committed:
Add Port to Input Fild

Due to that you can choose on which Port to listen for input. Usefull if you don't run logstash witout root User.

fabianfrz commented on opnsense.logstash.conf:
```diff
@@ -1,5 +1,6 @@
 input {
   syslog {
+    port => 5140
```
Can you comment it out so it is easy to add but by default it listens on the default syslog port?

superwinni2 committed:
Update opnsense.logstash.conf

Comment Out Port so everyone can choose self if he wants a special port or not.

superwinni2 commented:
Hopefully I did the right thing... I'm not very familiar with push and pull requests in Github

pir8g33k commented:
Hi [fabianfrz](https://github.com/fabianfrz) can we update this repo to work with the new update of opnsense please continue the projects
